deps/v8/src/heap/heap.h

```diff
@@ -1303,7 +1303,7 @@ class Heap {
 
   static size_t ComputeMaxSemiSpaceSize(uint64_t physical_memory) {
     const uint64_t min_physical_memory = 512 * MB;
-    const uint64_t max_physical_memory = 3 * static_cast<uint64_t>(GB);
+    const uint64_t max_physical_memory = 2 * static_cast<uint64_t>(GB);
```

Comment:
I did this because the tests were not passing. The tests were not passing because this function and related tests changed in V8 6.1 but we reverted the change in https://github.com/nodejs/node/pull/16413/commits/ba717d1ae1221bc9f6b2ff6ff3aaeaa77039a74b.

If we should keep the change with this version of the algorithm, please help me to update [the unit test](https://github.com/targos/node/blob/v8.x-v8-6.2/deps/v8/test/unittests/heap/heap-unittest.cc#L65-L79).